examples/nfts/NFTs.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity ^0.8.26;
+
+import {requireQuorum} from "pod-sdk/solidity-sdk/src/pod/Quorum.sol";
+
+interface IERC721Receiver {
+    // A contract must return this selector from onERC721Received to accept safe transfers. Any other value ⇒ revert.
+    function onERC721Received(
+        address operator, // msg.sender
+        address from, // previous owner
+        uint256 tokenId,
+        bytes calldata data
+    ) external returns (bytes4);
+}
+
+contract NFTs {
+    struct Token {
+        address owner;
+        string uri;
+        uint256 sequence;
+    }
+    mapping(uint256 => Token) public tokens;
+
+    event Minted(uint256 indexed tokenId, address indexed owner, string  uri);
+    event Transferred(uint256 indexed tokenId, address indexed from, address indexed to);
+
+    function mint(uint256 tokenId, string calldata uri) external {
+        requireQuorum(tokens[tokenId].owner == address(0), "already minted");
```

Comment:
It's better to have all NFTs initially belong to a particular address. So, if the token address is 0, then only the `tx.origin` can "mint", transferring it to another address.

Otherwise, there could be a race causing locks.